hare/config/config.go

```diff
@@ -10,5 +10,5 @@ type Config struct {
 }
 
 func DefaultConfig() Config {
-	return Config{800, 400, 200, time.Second * time.Duration(15)}
+	return Config{2, 1, 2, 500 *time.Millisecond}
```

Comment:
if we really want to start testing that so we should configure that now

Reply:
lets talk about it with @gavraz, I'll leave it as is for the meanwhile